rust/basic_ethereum/src/lib.rs

```diff
@@ -35,6 +35,57 @@ pub async fn ethereum_address(owner: Option<Principal>) -> String {
     wallet.ethereum_address().to_string()
 }
 
+#[derive(Debug, Deserialize)]
+#[allow(dead_code)]
```

Comment:
Because we only use the `result` field. The alternative would be to drop the `error` field but I think it's more informative to leave it in (you'd probably do something with this field in production code anyway).

Reply:
It would IMO be better to remove `#[allow(dead_code)]` and panic explicitly in case of an `error`. Note that `JsonRpcResult` should be an enum.

Reply:
Thanks a lot for the changes, much simpler now!